.github/workflows/build_and_test.yml

```
@@ -0,0 +1,380 @@
+            guarzo/wanderer-notifier:sha-${{ github.sha }}
+          build-args: |
+            APP_VERSION=${{ steps.version.outputs.full_version }}
+            VCS_REF=${{ github.sha }}
+            WANDERER_NOTIFIER_API_TOKEN=${{ secrets.NOTIFIER_API_TOKEN }}
+          cache-from: type=registry,ref=guarzo/wanderer-notifier:dev
+          cache-to: type=inline
+          docker_username: guarzo
+          docker_password: ${{ secrets.DOCKERHUB_PAT }}
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)